Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -118,6 +134,15 @@ public struct PreplannedListItemView: View {
         .font(.caption2)
         .foregroundStyle(.tertiary)
     }
+    
+    /// Sets the closure to call when the map selection changes.
+    public func onMapSelectionChanged(
+        perform action: @escaping (_ newMap: Map) -> Void
+    ) -> Self {
+        var copy = self
+        copy.onMapSelectionChanged = action
+        return copy
+    }
```

Comment:
Using a modifier for this doesn't really make a whole lot of sense for a value that is internal and always needed by the view. Instead, just pass the selection changed handler to the view via an initializer parameter.

Reply:
Done.